packages/governance/test/swingsetTests/committeeBinary/vat-voter.js

```diff
@@ -0,0 +1,36 @@
+// @ts-check
+
+import { E } from '@agoric/eventual-send';
+import { Far } from '@agoric/marshal';
+
+const build = async (log, zoe) => {
+  return Far('voter', {
+    createVoter: async (name, voteFacet) => {
+      return Far(`Voter ${name}`, {
+        voteBallot: (qName, ballot) => {
```

Comment:
If the voters are just casting ballots, why use invitations?

Reply:
The `qName` parameter is not used

Reply:
This PR doesn't include an election manager, which will add some of the crucial connections to support validation.

Reply:
I've added more legibility and more verification.  I wouldn't claim it's done. Some of the legibility comes from the connections to the election manager.